openreview/openreview.py

```diff
@@ -20,17 +21,17 @@ class OpenReviewException(Exception):
     pass
 
 class Client(object):
-
+    """
+    :param baseurl: URL to the host, example: https://openreview.net (should be replaced by 'host' name)
+    :type baseurl: str, optional
```

Comment:
defaults to environmental var OPENREVIEW_BASEURL

Reply:
Ok, I can add this